src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+  const messageTitle = document.createElement('h2');
+  const messageDescription = document.createElement('p');
+
+  message.setAttribute('class', `notification ${type}`);
+  messageTitle.setAttribute('class', 'title');
+
+  messageTitle.textContent = title;
+  messageDescription.textContent = description;
+
+  message.style.cssText = `position: fixed; top: ${posTop}px; right: ${posRight}px;`;
+
+  messageDescription.innerHTML =
+    'Message example. <br> Notification should contain title and description.';
```

Comment:
this seems redundant

Reply:
I have local erorrs, but mentor wrote commit and push